library/dd-sdk-android-session-replay/src/test/kotlin/com/datadog/android/sessionreplay/recorder/mapper/StringObfuscatorTest.kt

```diff
@@ -0,0 +1,310 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2016-Present Datadog, Inc.
+ */
+
+package com.datadog.android.sessionreplay.recorder.mapper
+
+import android.os.Build
+import com.datadog.android.sessionreplay.utils.ForgeConfigurator
+import com.datadog.tools.unit.annotations.TestTargetApi
+import com.datadog.tools.unit.extensions.ApiLevelExtension
+import fr.xgouchet.elmyr.Forge
+import fr.xgouchet.elmyr.junit5.ForgeConfiguration
+import fr.xgouchet.elmyr.junit5.ForgeExtension
+import java.util.LinkedList
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.jupiter.api.BeforeEach
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.extension.ExtendWith
+import org.junit.jupiter.api.extension.Extensions
+import org.mockito.junit.jupiter.MockitoExtension
+import org.mockito.junit.jupiter.MockitoSettings
+import org.mockito.quality.Strictness
+
+@Extensions(
+    ExtendWith(MockitoExtension::class),
+    ExtendWith(ForgeExtension::class),
+    ExtendWith(ApiLevelExtension::class)
+)
+@MockitoSettings(strictness = Strictness.LENIENT)
+@ForgeConfiguration(ForgeConfigurator::class)
+internal class StringObfuscatorTest {
+
+    lateinit var testedObfuscator: StringObfuscator
+
+    @BeforeEach
+    fun `set up`() {
+        testedObfuscator = StringObfuscator()
+    }
+
+    // region Android N and above
+
+    @Test
+    @TestTargetApi(Build.VERSION_CODES.N)
+    fun `M mask String W maskString(){string with newline, Android N}`(
```

Comment:
It is not `maskString`, but `obfuscate`. Same for other tests.

```suggestion
    fun `M mask String W obfuscate(){string with newline, Android N}`(
```